### PR TITLE
(refactor) O3-2709: Update placeholder camera icon used in registration form

### PR DESCRIPTION
**Pull request by psworld**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
In #1580 the icon that is used for image placeholder icon did not conform to the O3- style guidelines. 
I changed the placeholder icon to use `Camera Icon` available at [https://carbondesignsystem.com/guidelines/icons/library/](https://carbondesignsystem.com/guidelines/icons/library/) under **Technology** section and titled **Camera**

## Before
Screenshot 2024-01-19 at 09 00 55

## After
camera icon

## Related Issue
[https://openmrs.atlassian.net/browse/O3-2709](https://openmrs.atlassian.net/browse/O3-2709)

### CODE REVIEW

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss [hunk 1]
```diff
@@ -6,3 +6,12 @@
     background: none;
     cursor: pointer;
 }
+
+.placeholderIconContainer {
+    width: 64px;
+    height: 64px;
```
Comment (denniskigen) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss:12]: Could you amend this file to use the default indentation (2 tab widths)?

**Review by denniskigen — APPROVED** [review]
Thanks, @psworld!

**Comment by brandones** [comment]
Why didn't you just use the React component like we do with most of our icons? It's something like

```
import { Camera } from "@carbon/icons-react"

<Camera />
```

(but please refer to an actual usage in the codebase for the particulars).

**Comment by psworld** [comment]
> Why didn't you just use the React component like we do with most of our icons? It's something like
> 
> ```
> import { Camera } from "@carbon/icons-react"
> 
> <Camera />
> ```
> 
> (but please refer to an actual usage in the codebase for the particulars).

`placeholder.svg` was being used for the icon, I though that's how it should be. But now I am realizing that the custom `placeholder.svg` was probably being used for the Grey Rectangular Box. I totally missed it ! 

I have made the changes and used `<Camera />`  component.

Could you please guide on how should I push changes ? as this branch is already merged

1. Should I force push changes on this same branch ? amend the last commit ?
2. Create a new branch and push changes there ?

**Comment by denniskigen** [comment]
Please create a new PR , @psworld. Thanks for catching this, @brandones.

**Pull request merged**